source/board/mkit_dk_dongle_nrf5x.c

```diff
@@ -22,6 +22,8 @@
 #include "sam3u2c.h"
 #include "target_config.h"
 #include "util.h"
+#include "stdbool.h"
```

Comment:
Instead of adding the `stdbool.h` include here, add it to `flash_manager.h` where it's missing. Headers should (usually) be standalone and include all required includes within them.

Reply:
Sure, will incorporate it in the next commit.

Reply:
Committed.